docs/ota-client-guide/modules/ROOT/pages/garage-sign.adoc

```diff
@@ -0,0 +1,256 @@
+= garage-sign
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+xref:https://github.com/advancedtelematic/ota-tuf/tree/master/cli[garage-sign] is a tool for managing your OTA Connect software repository and securely signing software versions. It allows you to directly change, sign, and upload your xref:uptane.adoc[Uptane] metadata.
+
+.Why use `garage-sign`
+****
+`garage-sign` is useful in OTA Connect for managing the keys for signing your software. If your software image is not an OSTree image, you would need `garage-sign` to upload your xref:uptane.adoc[Uptane] metadata.
+
+Before using `garage-sign`, it is important to be familiar with the following two concepts.
+
+* *Offline keys*
++
+In a production environment, we recommend that you keep the keys for signing software versions offline. Therefore, the OTA Connect server will no longer sign software for you, instead, they will be signed locally by your machine. This maintains the security of the software because if someone gained access to your OTA Connect account, they will be able to send malicious software to your devices. However, if your keys are offline, your devices will not recognize the software as it will not have the correct credentials. This is where `garage-sign` becomes useful. For non-OSTree images, the `garage-sign` tool is used to manage and rotate your keys offline as well as sign your software.
+
+
+* *Local repository*
++
+A repository is a location for storing software packages. A local repository is a server that exists on your local machine such as your computer. When using the `garage-sign` tool, it is important to create a local repository on your machine that contains your credentials from OTA Connect, and your keys for signing metadata. With your keys existing locally on your machine, any changes you make to them will not directly affect your devices on OTA Connect. It also ensures the security of your keys during production.
+
+****
+
+
+== Prerequisites of `garage-sign`
+
+. Download the latest version of the `garage-sign` tool from link:https://tuf-cli-releases.ota.here.com/index.html[here].
+. Unzip the file you have downloaded to a location on your computer.
+. Change the directory of your terminal on the command line to that location.
+. Afterwards, open the `garage-sign` folder in your terminal. You can also print the help command to view the list of all possible commands with this tool.
++
+[source,bash]
+----
+cd garage-sign/
+bin/garage-sign --help
+----
+
+For more information, please refer to the link:https://github.com/advancedtelematic/ota-tuf/tree/master/cli[`garage-sign` README file].

```

Comment:
The whole damn point of this task is to improve on and replace that content. If you're linking to it for "more information", something has gone very wrong.